js/bundles/modules/parts/widgets-base.js

```diff
@@ -91,4 +89,9 @@ ui.CollectionWidget = require('../../../ui/collection/ui.collection_widget.edit'
 ui.dxDropDownEditor = require('../../../ui/drop_down_editor/ui.drop_down_editor');
 // Reports
 
+// Renovation
+ui.Button = require('../../../renovation/dist/button.j');
```

Comment:
This line rewrites classic ui.Button (line 24)

Reply:
Let's add `renovation` namespace

Reply:
We don't have conflicts here